README.md

```diff
@@ -155,6 +155,7 @@ Check our [Roadmap][roadmap] for upcoming planned integrations.
 - [How does Kev differ from Kompose?](docs/tutorials/how-kev-differs-from-kompose.md)
 - [Getting started with Kev](docs/tutorials/getting-started-with-kev.md)
 - [Iterate on the app with Kev and Skaffold](docs/tutorials/kev-dev-with-skaffold.md)
```

Comment:
Again ... I realise this is not part of this PR ... but maybe a rename ??
`Iterate on the app with Kev and Skaffold` -> `Develop and auto deploy with Kev and Skaffold`

Reply:
it's not only about deployment, as Skaffold takes care of build, push too. I think it can stay as is.

Reply:
Could change `Interate` -> `Develop` if that's any better.

Reply:
Let's go with `Develop`.